amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/raml/RamlDocumentParser.scala

```diff
@@ -614,21 +618,27 @@ abstract class RamlSpecParser(implicit ctx: RamlWebApiContext) extends WebApiBas
 
       // We parse the node as if it were a data shape, this will also check the closed node condition including the
       // annotation type facets
-      val maybeAnnotationType: Option[YMapEntry] = ast match {
-        case me: YMapEntry => Some(me)
-        case m: YMap       => Some(YMapEntry(YNode("annotationType"), YNode(m)))
-        case _             => None
+      // Only the value (map or scalar) is the shape. The key is the annotation name, and is not part of the type
+      val (maybeAnnotationType, name): (Option[YNode], Option[String]) = ast match {
```

Comment:
Could be moved to function.

Reply:
Y la verdad que seria mas prolijo, en ese match y en varias parts del codigo. Apunte a modificar el caso que necesitaba y no meter muchos cambios en estructura.